_posts/2020-07-16-omero-5-6-2.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: Release of OMERO.server 5.6.2 and OMERO.web 5.7.0
+intro-blurb: This release includes OMERO.server 5.6.2, OMERO.web 5.7.0, OMERO.py 5.7.1 and OMERO.iviewer 0.10.0
+---
+
+Today we are releasing OMERO.server 5.6.2 which fixes installation instructions for CentOS 7, 
+adds installation instructions for both OMERO.web and OMERO.server on CentOS 8 and Ubuntu 20.04,
+and finally drops support for Ubuntu 16.04 and removes the corresponding installation instructions.
+We are also releasing OMERO.web 5.7.0 and OMERO.py 5.7.1 which both contain some API additions.
+
+OMERO.server 5.6.2 runs in the same deployment environments as OMERO.server 5.6.0 and
+the Bio-Formats Memoizer cache will be invalidated.
+
+### Installing the Software:
+
+For full details of the changes with the OMERO 5.6 series see the
+[OMERO 5.6.0]({{ site.baseurl }}/2020/01/15/omero-5-6-0.html) release
+announcement. OMERO.server 5.6.2 is designed to be a small step beyond the 5.6.0 release
```

Comment:
I think this sentence is now redundant but I don't mind it. The point was to not inhibit people from adopting the security release.